requirements.txt

```diff
@@ -5,13 +5,16 @@ kiwisolver==1.1.0
 matplotlib==3.1.2
 numpy==1.18.1
 pandas==0.25.3
-pkg-resources==0.0.0
 pyparsing==2.4.6
 python-dateutil==2.8.1
 pytz==2019.3
-scikit-learn==0.22.1
+scikit-learn==0.22.2.post1
```

Comment:
What does `0.22.2.post1` have that `0.22.1` doesn't? I guess it isn't a problem if we are just adopting a later version of `sklearn`, but I wonder if that could introduce other unexpected problems.

Reply:
I do not know, this is the version that Brandon said to use. I have used it a lot. As long as you have the requirements.txt loaded correctly it works fine.

Reply:
This is the version being used by d3m right now, so this made sure we were compatible with them when we loaded our model into their ecosystem.